site/src/Sidecar.jsx

```diff
@@ -0,0 +1,171 @@
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import FlightTakeoffIcon from "@mui/icons-material/FlightTakeoff";
+
+function Sidecar({ open, setOpen, map, setVisibleTypes, setActiveThemes }) {
+  const handleParis = () => {
+    setVisibleTypes([
+      "division_area",
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "segment",
+      "connector",
+      "building",
+      "building_part",
+    ]);
+    setActiveThemes(["transportation"]);
+    map.ref.current.jumpTo({
+      center: [2.3417, 48.8552],
+      zoom: 11.73,
+      pitch: 12,
+      bearing: 15.3,
+    });
+  };
+
+  const handleNYC = () => {
+    setVisibleTypes([
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "building",
+      "building_part",
+    ]);
+    setActiveThemes(["buildings"]);
+    map.ref.current.jumpTo({
+      center: [-73.99768, 40.75332],
+      zoom: 14.22,
+      pitch: 60,
+      bearing: 60.6,
+    });
+  };
+  const handleWorld = () => {
+    setVisibleTypes([
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+    ]);
+    setActiveThemes(["base"]);
+    map.ref.current.jumpTo({
+      center: [10.3, 24.5],
+      zoom: 1.07,
+      bearing: 0,
+      pitch: 0,
+    });
+  };
+  const handleLondon = () => {
+    setVisibleTypes([
+      "division_area",
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "segment",
+      "building",
+      "building_part",
+      "place",
+    ]);
+    setActiveThemes(["places"]);
+    map.ref.current.jumpTo({
+      center: [-0.091217, 51.514511],
+      zoom: 16.02,
+      bearing: -24,
+      pitch: 50,
+    });
+  };
+  const handleBoston = () => {
+    setVisibleTypes([
+      "boundary",
+      "land",
+      "land_cover",
+      "land_use",
+      "water",
+      "infrastructure",
+      "segment",
+      "building",
+      "building_part",
+      "address",
+    ]);
+    setActiveThemes(["addresses"]);
+    map.ref.current.jumpTo({
+      center: [-71.065192, 42.353714],
+      zoom: 15.94,
+      bearing: 0,
+      pitch: 52,
+    });
+  };
+  return (
+    <>
+      {open ? (
+        <div className="sidecar-modal">
+          <div className="sidecar-box">
+            <div className="sidecar-landing">
+              <p className="sidecar-welcome">WELCOME</p>
+              <button
+                className="close-panel-button"
+                onClick={() => setOpen(false)}
+              >
+                <CloseIcon className="close-panel-icon" />
+              </button>
+            </div>
+            <div className="sidecar-header">Explore Overture</div>
+            <div className="sidecar-blurb">
+              Initially, we are focusing on layers for transportation, places,
```

Comment:
This text is redundant - they can see from the items we've included that we are focusing on these places! 

How about more of a welcome-y feel, like 'We've picked a few spots around the world you might be interested in seeing."